graylog2-server/src/main/java/org/graylog/events/notifications/types/EmailEventNotificationConfig.java

```diff
@@ -134,7 +166,8 @@ public static Builder create() {
                     .subject(DEFAULT_SUBJECT)
                     .emailRecipients(ImmutableSet.of())
                     .userRecipients(ImmutableSet.of())
-                    .bodyTemplate(DEFAULT_BODY_TEMPLATE);
+                    .bodyTemplate(DEFAULT_BODY_TEMPLATE)
+                    .htmlBodyTemplate(DEFAULT_HTML_BODY_TEMPLATE);
```

Comment:
This should be an empty String by default. We don't want to automatically upgrade existing notifications to HTML.

Reply:
```suggestion
                    .htmlBodyTemplate("");
```